src/utils/addresses.ts

```diff
@@ -63,7 +71,13 @@ export const storeAddressMetadataOfAccount = (accountName: string, index: number
     Object.assign(existingAddressMetadata, settings)
   }
   console.log(`🟠 Storing address index ${index} metadata locally`)
-  localStorage.setItem(constructMetadataKey(accountName), JSON.stringify(addressesMetadata))
+  localStorage.setItem(
+    constructMetadataKey(accountName),
+    JSON.stringify({
+      version: latestUserDataVersion,
+      encryptedSettings: encrypt(mnemonic, JSON.stringify(addressesMetadata))
```

Comment:
Why are we using the mnemonic to encrypt the address metadata and not the password?